### ISSUE TITLE
use a log module instead of homebrew

### ISSUE THREAD

**Issue by mafintosh** [mafintosh/hms#8]

**Comment by gausby** [comment]
Any ideas or preferences on which log module to use?

**Comment by mafintosh** [comment]
anything that is simple and writes single line messages with date stamps.

**Comment by gausby** [comment]
How creating a module like this:
```js
function GenerateLogger (namespace) {
  namespace = '[' + namespace + ']'

  function now () {
    return (new Date).toISOString()
  }

  return function (message) {
    message = [now(), namespace, message].filter(Boolean).join(' ')
    console.log.apply(null, arguments)
  }
}

var log = GenerateLogger('dock');

log('single argument')
// 2015-01-30T10:34:51.691Z [dock] single argument

log('multiple arguments', 'one more')
// 2015-01-30T10:34:51.707Z [dock] multiple arguments one more

log(undefined, 'undefined as the first argument')
// 2015-01-30T10:34:51.708Z [dock] undefined as the first argument
```

**Comment by gausby** [comment]
How about this piece of art? https://github.com/gausby/log-with-namespace-and-date-stamp

**Comment by mafintosh** [comment]
fixed by #39
